### ISSUE TITLE
How to start Chrome with a specific language?

### ISSUE THREAD

**Issue by bennyn** [karma-runner/karma-chrome-launcher#70]
Is there a possibility to tell Karma that it should always start a Chrome instance with an English locale?

My current Karma configuration is this:

``` javascript
// available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
browsers: [process.env.TRAVIS ? 'Chrome_travis_ci' : 'Chrome'],
customLaunchers: {
  Chrome_travis_ci: {
    base: 'Chrome',
    flags: ['--no-sandbox']
  }
}
```

**Comment by rostacik** [comment]
Hello sir.
Not sure if still in need but I have tested running Chrome with switch lang like found here :
http://peter.sh/experiments/chromium-command-line-switches/#lang

Works for me, but please consider also this information :
https://developer.chrome.com/extensions/i18n#locales-testing

TLDR - you need to close all other Chrome instances (with same user as your run tests agains). If your force some language you will also see that UI of Chrome changed.

I tested on my dev box under Windows 10, not sure how Travis will behave.

**Comment by rogeriopvl** [comment]
Closing due to inactivity.

**Comment by bennyn** [comment]
@rogeriopvl Who (or what) is inactive? The development of Karma Chrome Launcher?

**Comment by rogeriopvl** [comment]
@bennyn the issue.
